src/constants/constants.js

```diff
@@ -75,10 +66,35 @@ const PostsTypes = {
     REMOVE_FROM_CHANNEL: 'system_remove_from_channel'
 };
 
+const Permissions = {
+    PERMISSIONS_ALL: 'all',
+    PERMISSIONS_CHANNEL_ADMIN: 'channel_admin',
+    PERMISSIONS_TEAM_ADMIN: 'team_admin',
+    PERMISSIONS_SYSTEM_ADMIN: 'system_admin',
```

Comment:
cool let me change them, this was actually a copy/paste from the webapp, but it makes total sense